engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -300,42 +308,31 @@ struct hammer_of_the_righteous_t : public paladin_melee_attack_t
 // TODO(mserrano): fix judgment
 struct judgment_prot_t : public judgment_t
 {
-  timespan_t sotr_cdr; // needed for sotr interaction for protection
-
   judgment_prot_t( paladin_t* p, const std::string& options_str ) :
     judgment_t( p, options_str )
   {
     cooldown -> charges += as<int>( p -> talents.crusaders_judgment -> effectN( 1 ).base_value() );
-    cooldown -> duration *= 1.0 + p -> spec.protection_paladin -> effectN( 4 ).percent();
-
-    base_multiplier *= 1.0 + p -> spec.protection_paladin -> effectN( 12 ).percent();
```

Comment:
Looking at spelldata, this modifier is still there (although in effect#13)

Reply:
Yeah, but I get the right numbers without it. Could some automated system be taking it into account?

Reply:
It's getting auto-applied in https://github.com/simulationcraft/simc/blob/8b90961d75ca46160701699bac8ff88dda2aff92/engine/class_modules/paladin/sc_paladin.cpp#L2718